# INSTRUCTION
float64 roll
float64 pitch
float64 yaw
float64 altitude
---
# RESULT
bool    success
string  status
---
# FEEDBACK
float64 elapsed
float64 travelled
float64 remaining